portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,33 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+/**
+ * Cycles through images automatically.
+ */
+function slideShow(imgIndex = 0) {
+
+  var slides = document.getElementsByClassName('mySlides');
```

Comment:
Use const or let for variable declarations: https://google.github.io/styleguide/jsguide.html#features-use-const-and-let